src/main/java/org/fogbowcloud/sebal/SEBALHelper.java

```diff
@@ -115,14 +115,14 @@ public static int centralMeridian(int zoneNumber) throws Exception {
 			StringTokenizer tokenizedResponse = new StringTokenizer(responseStr, "\n");
 			while (tokenizedResponse.hasMoreTokens()) {
```

Comment:
I think this method do too many things. You should split it in 2 or 3 methods. At least, one for fetching remote content (with proper error handling code) and other for parsing the fetched string,